modules/grandpa/src/benchmarking.rs

```diff
@@ -201,81 +141,4 @@ benchmarks_instance_pallet! {
 		assert_eq!(<BestFinalized<T, I>>::get(), expected_hash);
 		assert!(<ImportedHeaders<T, I>>::contains_key(expected_hash));
 	}
-
-	// Here we want to find out the overheaded of looking through consensus digests found in a
-	// header. As the number of logs in a header grows, how much more work do we require to look
-	// through them?
-	//
-	// Note that this should be the same for looking through scheduled changes and forces changes,
-	// which is why we only have one benchmark for this.
-	find_scheduled_change {
-		// Not really sure what a good bound for this is.
-		let n in 1..1000;
-
-		let mut logs = vec![];
-		for i in 0..n {
-			// We chose a non-consensus log on purpose since that way we have to look through all
-			// the logs in the header
-			logs.push(sp_runtime::DigestItem::Other(vec![]));
-		}
-
-		let mut header: BridgedHeader<T, I> = bp_test_utils::test_header(Zero::zero());
-		let digest = header.digest_mut();
-		*digest = sp_runtime::Digest {
-			logs,
-		};
-
-	}: {
-		crate::find_scheduled_change(&header)
-	}
-
-	// What we want to check here is how long it takes to read and write the authority set tracked
-	// by the pallet as the number of authorities grows.
-	read_write_authority_sets {
-		// The current max target number of validators on Polkadot/Kusama
-		let n in 1..1000;
-
-		let mut authorities = vec![];
-		for i in 0..n {
-			authorities.push((ALICE, 1));
-		}
-
-		let authority_set = bp_header_chain::AuthoritySet {
-			authorities: authorities.iter().map(|(id, w)| (AuthorityId::from(*id), *w)).collect(),
-			set_id: 0
-		};
-
-		<CurrentAuthoritySet<T, I>>::put(&authority_set);
-
-	}: {
-		let authority_set = <CurrentAuthoritySet<T, I>>::get();
-		<CurrentAuthoritySet<T, I>>::put(&authority_set);
-	}
-}
-
-#[cfg(test)]
```

Comment:
That's not true - `verify` blocks are executed after every benchmark run. The tests in this module, however are never executed because we never ran cargo test with features=runtime-benchmarks on CI.

Reply:
> The tests in this module, however are never executed because we never ran cargo test with features=runtime-benchmarks on CI.

🤔

Okay, well nevermind then, haha